src/styles/main.scss

```diff
@@ -1,4 +1,4 @@
-.main {
-  background: green;
-  padding: 10px;
+#app {
```

Comment:
global styles could be applied with `createGlobalStyle` from `styled-components`